tests/integration-tests/tests/storage/test_ebs.py

```diff
@@ -24,9 +25,13 @@
 @pytest.mark.instances(["c4.xlarge", "c5.xlarge"])
 @pytest.mark.schedulers(["sge"])
 @pytest.mark.usefixtures("region", "os", "instance")
-def test_ebs_single(scheduler, pcluster_config_reader, clusters_factory):
+def test_ebs_single(scheduler, pcluster_config_reader, clusters_factory, kms_key_factory):
     mount_dir = "ebs_mount_dir"
-    cluster_config = pcluster_config_reader(mount_dir=mount_dir)
+    kms_key_id = kms_key_factory.create_kms_key(scheduler)
```

Comment:
Are there checks we can add to make sure the volume was encrypted using the expected key? One idea is to use the boto3's `describe_volumes` to check whether the volume's `Encrypted` attribute is set to `True`.

Reply:
Good idea! Done!